src/seedu/addressbook/ui/TextUi.java

```diff
@@ -18,65 +17,62 @@
  */
 public class TextUi {
 
-    /**
-     * A decorative prefix added to the beginning of lines printed by AddressBook
-     */
-    public static final String LINE_PREFIX = "|| ";
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
 
-    /**
-     * A platform independent line separator.
-     */
-    public static final String LS = System.lineSeparator();
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
 
-    public static final String DIVIDER = "===================================================";
+    private static final String DIVIDER = "===================================================";
```

Comment:
I'm actually curious why we make it private, when it's a final immutable object there's no security concerns right?